src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -520,6 +520,8 @@
     <Compile Include="$(BclSourcesRoot)\System\Diagnostics\Eventing\EventPipe.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Diagnostics\Eventing\EventPipeEventProvider.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Diagnostics\Eventing\EventPipeMetadataGenerator.cs" />
+    <Compile Include="$(IntermediateOutputPath)..\eventing\DotNETRuntimeEventSource.cs" />
```

Comment:
Is `build.cmd windowsmscorlib` on clean repo still going to work with this?

I know that it is used a lot by people contributing to corelib.

Reply:
Also check that opening System.Private.CoreLib.sln in Visual Studio still works.

Reply:
It works now.  I had originally planned support for this, but hadn't sufficiently tested to ensure that directories got created properly.  I've just pushed a fix.

The solution is also openable - it lists the files there but if you try to open them VS just says it couldn't find them.  There is no error dialog or anything that really draws attention to them otherwise.  This is what I see when I open System.Private.CoreLib.sln in VS2017 before the files have been generated:

![image](https://user-images.githubusercontent.com/6210322/40081827-fdce3b2c-5843-11e8-9b70-24ca442e1177.png)